bootstrap/003_fake_account_usage.sql

```diff
@@ -241,3 +241,132 @@ null::TEXT as "CLIENT_BUILD_ID",
+null::BOOLEAN as "DELETED",
+null::TIMESTAMP_LTZ as "TABLE_CREATED",
+null::TIMESTAMP_LTZ as "TABLE_DROPPED",
+null::TIMESTAMP_LTZ as "TABLE_ENTERED_FAILSAFE",
+null::TIMESTAMP_LTZ as "SCHEMA_CREATED",
+null::TIMESTAMP_LTZ as "SCHEMA_DROPPED",
+null::TIMESTAMP_LTZ as "CATALOG_CREATED",
+null::TIMESTAMP_LTZ as "CATALOG_DROPPED",
+null::TEXT as "COMMENT",
+null::NUMBER as "INSTANCE_ID"
+;
+
+CREATE VIEW ACCOUNT_USAGE.HYBRID_TABLES IF NOT EXISTS AS SELECT
+null::NUMBER as "ID",
+null::TEXT as "NAME",
+null::NUMBER as "SCHEMA_ID",
+null::TEXT as "SCHEMA_NAME",
+null::NUMBER as "DATABASE_ID",
+null::TEXT as "DATABASE_NAME",
+null::TEXT as "OWNER",
+null::NUMBER as "ROW_COUNT",
+null::NUMBER as "BYTES",
+null::NUMBER as "RETENTION_TIME",
+null::TIMESTAMP_LTZ as "CREATED",
+null::TIMESTAMP_LTZ as "LAST_ALTERED",
+null::TIMESTAMP_LTZ as "DELETED",
+null::TEXT as "COMMENT",
+null::TEXT as "OWNER_ROLE_TYPE"
+;
+
+CREATE VIEW ACCOUNT_USAGE.HYBRID_TABLES_USAGE_HISTORY IF NOT EXISTS AS SELECT
+null::TEXT as "OBJECT_TYPE",
+null::NUMBER as "OBJECT_ID",
+null::TEXT as "OBJECT_NAME",
+null::TIMESTAMP_LTZ as "START_TIME",
+null::TIMESTAMP_LTZ as "END_TIME",
+null::NUMBER as "CREDITS_USED"
+;
+
+CREATE VIEW ACCOUNT_USAGE.MATERIALIZED_VIEW_REFRESH_HISTORY IF NOT EXISTS AS SELECT
+null::TIMESTAMP_LTZ as "START_TIME",
+null::TIMESTAMP_LTZ as "END_TIME",
+null::TEXT as "CREDITS_USED",
+null::NUMBER as "TABLE_ID",
+null::TEXT as "TABLE_NAME",
+null::NUMBER as "SCHEMA_ID",
+null::TEXT as "SCHEMA_NAME",
+null::NUMBER as "DATABASE_ID",
+null::TEXT as "DATABASE_NAME"
+;
+
+
+CREATE VIEW ACCOUNT_USAGE.OBJECT_DEPENDENCIES IF NOT EXISTS AS SELECT
+null::TEXT as "REFERENCED_DATABASE",
+null::TEXT as "REFERENCED_SCHEMA",
+null::TEXT as "REFERENCED_OBJECT_NAME",
+null::NUMBER as "REFERENCED_OBJECT_ID",
+null::TEXT as "REFERENCED_OBJECT_DOMAIN",
+null::TEXT as "REFERENCING_DATABASE",
+null::TEXT as "REFERENCING_SCHEMA",
+null::TEXT as "REFERENCING_OBJECT_NAME",
+null::NUMBER as "REFERENCING_OBJECT_ID",
+null::TEXT as "REFERENCING_OBJECT_DOMAIN",
+null::TEXT as "DEPENDENCY_TYPE"
+;
+
+CREATE VIEW ACCOUNT_USAGE.TAG_REFRENCES IF NOT EXISTS AS SELECT
+null::TEXT as "TAG_DATABASE",
+null::TEXT as "TAG_SCHEMA",
+null::NUMBER as "TAG_ID",
+null::TEXT as "TAG_NAME",
+null::TEXT as "TAG_VALUE",
+null::TEXT as "OBJECT_DATABASE",
+null::TEXT as "OBJECT_SCHEMA",
+null::NUMBER as "OBJECT_ID",
+null::TEXT as "OBJECT_NAME",
+null::TIMESTAMP_LTZ as "OBJECT_DELETED",
+null::TEXT as "DOMAIN",
+null::NUMBER as "COLUMN_ID",
+null::TEXT as "COLUMN_NAME"
+;
+
+CREATE VIEW ACCOUNT_USAGE.TAGS IF NOT EXISTS AS SELECT
+null::NUMBER as "TAG_ID",
+null::TEXT as "TAG_NAME",
+null::NUMBER as "TAG_SCHEMA_ID",
+null::TEXT as "TAG_SCHEMA",
+null::NUMBER as "TAG_DATABASE_ID",
+null::TEXT as "TAG_DATABASE",
+null::TEXT as "TAG_OWNER",
+null::TEXT as "TAG_COMMENT",
+null::TIMESTAMP_LTZ as "CREATED",
+null::TIMESTAMP_LTZ as "LAST_ALTERED",
+null::TIMESTAMP_LTZ as "DELETED",
+null::ARRAY as "ALLOWED_VALUES",
```

Comment:
```suggestion
null::VARIANT as "ALLOWED_VALUES",
```

If we want to go off the definition for the view in Snowflake, should be `VARIANT`.